src/bin/scheduler/setup/directories.rs

```diff
@@ -20,6 +18,10 @@ pub fn setup(
     global_config: &GlobalConfig,
     plans: Vec<Plan>,
 ) -> Result<(Vec<Plan>, Vec<SetupFailure>), Terminate> {
+    let ownership_setter = OwnershipSetter::make_for_current_user()?;
```

Comment:
OTI: I appreciate the small commit size, they don't have to be this small though haha